lecture-notes/3-bld.md

```diff
@@ -418,20 +420,284 @@ order:
   provide the names of all of the object files to GCC so they can be linked
   together. It is *not* used in the `.o` rules because generally you do not
   pass `.h` files to the compiler (they are `include`d).
-* `$<` <!-- TODO -->
+* `$<` refers to the name of the first prerequisite. You can infer from this
+  that generally people write C Makefiles in a way that makes this variable
+  useful:
+	```make
+	main.o: main.c main.h
+		gcc -c $<
+	```
+  Both `main.c` and `main.h` are prerequisites for `main.o`, but because
+  `main.c` is listed as the first prerequisite, it can be referred to by `$<`
+  in the build recipe.
```

Comment:
Maybe a footnote explaining why the .h files don't need to be listed in the command explicitly?